src/sentry/stacktraces/processing.py

```diff
@@ -33,6 +33,9 @@ def __eq__(self, other: object) -> bool:
     def __ne__(self, other: object) -> bool:
         return self is not other
 
+    def get_frames(self) -> Sequence[Dict[str, Any]] | None:
+        return get_path(self.stacktrace, "frames", filter=True, default=())
```

Comment:
This call was happening all throughout this file.

Reply:
Perhaps this is a moot point given other refactors you have coming up, but if you made this a helper function (rather than a `StacktraceInfo` method), you could use it in places like `_append_stacktrace` also.